test/externalTests/rayTrace.js

```diff
@@ -3,11 +3,11 @@ const assert = require('assert')
 module.exports = () => (bot, done) => {
   const { position } = bot.entity
   bot.lookAt(position.offset(0, 3, 0), true, () => {
-    const block = bot.blockInSight()
-    assert.strictEqual(block, undefined)
+    const block = bot.blockAtCursor()
+    assert.strictEqual(block, null)
 
     bot.lookAt(position.offset(0, -3, 0), true, () => {
-      const block = bot.blockInSight()
+      const block = bot.blockAtCursor()
```

Comment:
Can we have the old test as it was and add a new one below? Just to be sure, that new `blockInSight()` logic stays 1:1 (as the `undefined` changes to `null`)

Reply:
done